### PR TITLE
WSC Configuration

### PR DESCRIPTION
**Pull request by DavidLiedle**
Ping @sunpietro

### CODE REVIEW

File: installation/run.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ / _ \ / /  |  _| | '_ \| __/ _ \ '__| '_ \| '__| / __|/ _ \\
+|  __// /_  | |___| | | | ||  __/ |  | |_) | |  | \__ \  __/
+ \___/____| |_____|_| |_|\__\___|_|  | .__/|_|  |_|___/\___|
+                                     |_|
+"
+echo -n "Running composer installation..."
+composer install -n
+echo "..DONE!"
+echo "Obtaining the initial database..."
+php -d memory_limit=-1 app/console ezplatform:install --env prod clean
```
Comment (sunpietro) [inline installation/run.sh:19]: Please, update this line to: `php app/console ezplatform:install demo`

**Comment by sunpietro** [comment]
The only thing I'm missing in installation scripts is `php app/console cache:clear --env=prod; php app/console assets:install --env=prod --symlink; php app/console assetic:dump --env=prod`
It will generate assets.

**Comment by emodric** [comment]
@sunpietro That will all be taken care of by Composer.

**Comment by sunpietro** [comment]
@emodric of course, but it won't create any symlinks as far as I know. So whenever you change your JS code, you'll have to rebuild and reinstall assets

**Comment by emodric** [comment]
@sunpietro It will. `"symfony-assets-install": "relative"` in `composer.json` makes sure that `assets:install` creates the symlinks.

**Comment by emodric** [comment]
@sunpietro P.S. I see that you didn't include https://github.com/ezsystems/wsc-rest-demo-block in the repo. Don't forget to include it :)

**Pull request merged**